Playground/RxSwiftExtPlayground.playground/Pages/apply.xcplaygroundpage/Contents.swift

```diff
@@ -44,4 +44,36 @@ example("apply a transformation") {
     })
 }
 
+/*:
+ ## apply
+
+ The `apply` operator takes a transformation function `(Single) -> Single` and applies it to the stream. The purpose of this operator is to provide syntactic sugar for applying multiple operators to the stream, while preserving the chaining operator structure of Rx.
+
+ */
+
+func addOne(input: Single<Int>) -> Single<String> {
```

Comment:
Hey, sorry for nit-picking but this isn't exactly what I mean. Just adding a few example using `Single` in the existing sample on top would be enough. No need for an additional title and more code blocks :)